core/scripts/ocr2vrf/util.go

```diff
@@ -220,6 +220,30 @@ func setProducer(e helpers.Environment, vrfCoordinatorAddr, vrfBeaconAddr string
 	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
 }
 
+func createSubscription(e helpers.Environment, vrfCoordinatorAddr string) {
+	coordinator := newVRFCoordinator(common.HexToAddress(vrfCoordinatorAddr), e.Ec)
+
+	tx, err := coordinator.CreateSubscription(e.Owner)
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func addConsumer(e helpers.Environment, vrfCoordinatorAddr, consumerAddr string, subId *big.Int) {
+	coordinator := newVRFCoordinator(common.HexToAddress(vrfCoordinatorAddr), e.Ec)
+
+	tx, err := coordinator.AddConsumer(e.Owner, subId.Uint64(), common.HexToAddress(consumerAddr))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func setPayees(e helpers.Environment, vrfBeaconAddr string, transmitters, payees []common.Address) {
```

Comment:
Ditto here, should we call this during the setup to automatically set the payees?

Reply:
yep :+1: